doc: changelog for 2.9.99 (#19241)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

.. _ncs_release_notes_changelog:

> Changelog for |NCS| v2.9.99

Reviewer: ```suggestion
Changelog for |NCS| v2.8.99
```

************

Known issues are only tracked for the latest official release.

> See `known issues` for the list of issues valid for the latest release.

Reviewer 2: ```suggestion
See :ref:`known_issues` for the list of issues valid for the latest release.
```

> See `known issues for nRF Connect SDK v2.9.0`_ for the list of issues valid for the latest release.

Reviewer: ```suggestion
See :ref:`known_issues` for the list of issues valid for the latest release.
```

This also fixes the doc build.
